lighthouse-core/test/config/config-test.js

```diff
@@ -230,7 +223,7 @@ describe('Config', () => {
             'ViewportDimensions', // from gatherer
           ],
           __internalOptionalArtifacts: [
-            'SourceMaps', // Not in the config.
+            'SourceMapsBlah', // Not in the config.
```

Comment:
looks like this is exactly the same test as on line 151?

somehow these got repeated three times :)

Reply:
> somehow these got repeated three times :)

`does not throw when an audit requests an optional artifact with no gatherer supplying it` is still in there one extra time and `should keep optional artifacts in gatherers after filter` test is repeated three times :)